src/components/widgets/status/StatusTab.vue

```diff
@@ -2,9 +2,8 @@
   <div>
     <!-- Only show the linear progress for mdAndDown -->
     <v-progress-linear
-      v-if="
-        progressVisible &&
-          $vuetify.breakpoint.mdAndDown
+      v-if="progressVisible &&
+        $vuetify.breakpoint.mdAndDown
```

Comment:
LGTM functionality wise now, but I'd prefer it if unrelated lines weren't changed - looks like these were auto-formatted? Applies to https://github.com/fluidd-core/fluidd/pull/1252/files#diff-8014f06344747bd0e9f2beb41bbca3f9938e86a5228dbdb110ece4fb4acc3860R52-R139 too

Reply:
oops was trying to fix the other .css formatting will roll it back.